origin.spec

```diff
@@ -0,0 +1,419 @@
+install -m 0644 rel-eng/origin-node.sysconfig %{buildroot}%{_sysconfdir}/sysconfig/%{name}-node
+install -d -m 0755 %{buildroot}%{_prefix}/lib/tuned/%{name}-node-{guest,host}
+install -m 0644 tuned/origin-node-guest/tuned.conf %{buildroot}%{_prefix}/lib/tuned/%{name}-node-guest/tuned.conf
+install -m 0644 tuned/origin-node-host/tuned.conf %{buildroot}%{_prefix}/lib/tuned/%{name}-node-host/tuned.conf
+install -d -m 0755 %{buildroot}%{_mandir}/man7
+install -m 0644 tuned/man/tuned-profiles-origin-node.7 %{buildroot}%{_mandir}/man7/tuned-profiles-%{name}-node.7
+
+mkdir -p %{buildroot}%{_sharedstatedir}/%{name}
+mkdir -p %{buildroot}%{_sharedstatedir}/origin
+
+
+# Install sdn scripts
+install -d -m 0755 %{buildroot}%{kube_plugin_path}
+install -d -m 0755 %{buildroot}%{_unitdir}/docker.service.d
+install -p -m 0644 rel-eng/docker-sdn-ovs.conf %{buildroot}%{_unitdir}/docker.service.d/
+pushd _thirdpartyhacks/src/%{sdn_import_path}/ovssubnet/controller/kube/bin
+   install -p -m 755 openshift-ovs-subnet %{buildroot}%{kube_plugin_path}/openshift-ovs-subnet
+   install -p -m 755 openshift-sdn-kube-subnet-setup.sh %{buildroot}%{_bindir}/openshift-sdn-kube-subnet-setup.sh
+popd
+pushd _thirdpartyhacks/src/%{sdn_import_path}/ovssubnet/controller/multitenant/bin
+   install -p -m 755 openshift-ovs-multitenant %{buildroot}%{_bindir}/openshift-ovs-multitenant
+   install -p -m 755 openshift-sdn-multitenant-setup.sh %{buildroot}%{_bindir}/openshift-sdn-multitenant-setup.sh
+popd
+install -d -m 0755 %{buildroot}%{_unitdir}/%{name}-node.service.d
+install -p -m 0644 rel-eng/openshift-sdn-ovs.conf %{buildroot}%{_unitdir}/%{name}-node.service.d/openshift-sdn-ovs.conf
+
+# Install bash completions
+install -d -m 755 %{buildroot}%{_sysconfdir}/bash_completion.d/
+install -p -m 644 rel-eng/completions/bash/* %{buildroot}%{_sysconfdir}/bash_completion.d/
+# Generate atomic-enterprise bash completions
+%{__sed} -e "s|openshift|atomic-enterprise|g" rel-eng/completions/bash/openshift > %{buildroot}%{_sysconfdir}/bash_completion.d/atomic-enterprise
+
+%files
+%defattr(-,root,root,-)
+%doc README.md LICENSE
+%{_bindir}/oc
+%{_bindir}/openshift
+%{_bindir}/openshift-router
+%{_bindir}/openshift-deploy
+%{_bindir}/openshift-sti-build
+%{_bindir}/openshift-docker-build
+%{_bindir}/origin
+%{_bindir}/atomic-enterprise
+%{_bindir}/oadm
+%{_bindir}/kubectl
+%{_bindir}/kubernetes
+%{_bindir}/kubelet
+%{_bindir}/kube-proxy
+%{_bindir}/kube-apiserver
+%{_bindir}/kube-controller-manager
+%{_bindir}/kube-scheduler
+%{_sharedstatedir}/%{name}
+%{_sysconfdir}/bash_completion.d/*
+%dir %config(noreplace) %{_sysconfdir}/origin
+
+%pre
+# If /etc/openshift exists symlink it to /etc/origin
+if [ -d "%{_sysconfdir}/openshift" ]; then
+  ln -s %{_sysconfdir}/openshift %{_sysconfdir}/origin
+fi
+
+%files master
+%defattr(-,root,root,-)
+%{_unitdir}/%{name}-master.service
+%config(noreplace) %{_sysconfdir}/sysconfig/%{name}-master
```

Comment:
This is producing a different file than what is in the service file

```
$ rpm -qpl /tito/x86_64/atomic-openshift-master-0.0.1-0.git.5135.3e30cd3.el7ose.x86_64.rpm | grep sysconfig
/etc/sysconfig/atomic-openshift-master
$
```

Currently the ```atomic-openshift-master.service``` has:

```
...
EnvironmentFile=/etc/sysconfig/origin-master
...
```